lib/runPrettier.js

```diff
@@ -0,0 +1,61 @@
+const { spawn } = require('child_process');
+const path = require('path');
+
+const cwd = process.cwd();
+
+const filePattern = 'src/**/*.js';
+
+function runPrettier({ write, listDifferent, config }) {
+  const prettierConfig = config || {};
+
+  const prettierPath = path.join(
+    path.resolve('node_modules/prettier'),
```

Comment:
Might be safer to use a path relative to `require.resolve`, since AFAIK there's technically no guarantees where Prettier will be on the file system.